modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/SqlTransactionControlParserTest.java

```diff
@@ -62,14 +61,7 @@ public void testCommit() {
         expectUnparsed(node, sql);
     }
 
-    private static SqlNode parseStatement(String sqlStmt) {
-        return IgniteSqlParser.parse(sqlStmt, StatementParseResult.MODE).statement();
-    }
-
-    private static void expectUnparsed(SqlNode node, String expected) {
-        SqlPrettyWriter writer = new SqlPrettyWriter();
-        node.unparse(writer, 0, 0);
-
-        assertEquals(expected, writer.toString(), "Unparse does not match");
+    private SqlNode parseStatement(String sqlStmt) {
```

Comment:
Maybe it's worth removing this method completely?

Reply:
Sure, done.